src/proxy-ui-api/frontend/src/views/GenerateCertificateSignRequest/GenerateCertificateSignRequest.vue

```diff
@@ -0,0 +1,141 @@
+
+<template>
+  <div class="view-wrap">
+    <subViewTitle class="view-title" :title="$t('csr.generateCsr')" :showClose="false" />
+    <v-stepper :alt-labels="true" v-model="e1" class="stepper noshadow">
+      <v-stepper-header class="noshadow">
+        <v-stepper-step :complete="e1 > 1" step="1">{{$t('csr.csrDetails')}}</v-stepper-step>
+        <v-divider></v-divider>
+        <v-stepper-step :complete="e1 > 2" step="2">{{$t('csr.generateCsr')}}</v-stepper-step>
+      </v-stepper-header>
+
+      <v-stepper-items class="stepper-content">
+        <!-- Step 1 -->
+        <v-stepper-content step="1">
+          <WizardPageCsrDetails @cancel="cancel" @done="save" />
+        </v-stepper-content>
+        <!-- Step 2 -->
+        <v-stepper-content step="2">
+          <WizardPageGenerateCsr @cancel="cancel" @done="done" />
+        </v-stepper-content>
+      </v-stepper-items>
+    </v-stepper>
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import { mapGetters } from 'vuex';
+import HelpIcon from '@/components/ui/HelpIcon.vue';
+import LargeButton from '@/components/ui/LargeButton.vue';
+import SubViewTitle from '@/components/ui/SubViewTitle.vue';
+import WizardPageCsrDetails from './WizardPageCsrDetails.vue';
+import WizardPageGenerateCsr from './WizardPageGenerateCsr.vue';
+
+import { Key, Token } from '@/types';
+import { RouteName, UsageTypes } from '@/global';
+import { saveAsFile } from '@/util/helpers';
+import * as api from '@/util/api';
+
+export default Vue.extend({
+  components: {
+    HelpIcon,
+    LargeButton,
+    SubViewTitle,
+    WizardPageCsrDetails,
+    WizardPageGenerateCsr,
+  },
+  props: {
+    keyId: {
+      type: String,
+      required: true,
+    },
+  },
+  data() {
+    return {
+      e1: 0,
```

Comment:
I wish there was a more descriptive name for this variable. Or is this really more or less the standard way to be used in this context?

Reply:
Fixed